packages/docs/src/components/layout.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Header, Footer } from "@nice-digital/global-nav";
+import { Grid, GridItem } from "@nice-digital/nds-grid";
+import "./main.scss";
+
+interface ILayout {
+	children: React.ReactNode;
+}
+
+const Layout = (props: ILayout): React.ReactElement => (
```

Comment:
Can you write a simple test for a component/page within the gatsby site just to prove that running tests from the root picks up files in here?

Reply:
@ediblecode Is just a `todo` test okay for now?